src/saga/adaptors/slurm/slurm_job.py

```diff
@@ -473,6 +474,12 @@ def _job_run (self, jd) :
         if jd.attribute_exists("total_physical_memory"):
             job_memory = jd.total_physical_memory
 
+        if jd.attribute_exists("candidate_hosts"):
+            if isinstance(jd.candidate_hosts, (tuple, list)):
```

Comment:
Any particular reason why you also support a tuple? The spec says list and this is also what we generally support in SAGA.